src/sentry/replays/usecases/query/__init__.py

```diff
@@ -43,6 +43,46 @@
 from sentry.replays.usecases.query.fields import ComputedField, TagField
 from sentry.utils.snuba import raw_snql_query
 
+VIEWED_BY_ME_KEY_ALIASES = ["viewed_by_me", "seen_by_me"]
+NULL_VIEWED_BY_ID_VALUE = 0  # default value in clickhouse
+
+
+def handle_viewed_by_me_filters(
+    search_filters: Sequence[SearchFilter | str | ParenExpression], request_user_id: int | None
+) -> Sequence[SearchFilter | str | ParenExpression]:
+    """Translate "viewed_by_me" as it's not a valid Snuba field, but a convenience alias for the frontend"""
+    if request_user_id is None:
+        request_user_id = NULL_VIEWED_BY_ID_VALUE
+
+    new_filters = []
+    for search_filter in search_filters:
+        if (
+            not isinstance(search_filter, SearchFilter)
+            or search_filter.key.name not in VIEWED_BY_ME_KEY_ALIASES
+        ):
+            new_filters.append(search_filter)
+            continue
+
+        # since the value is boolean, negations (!) are not allowed
+        if search_filter.operator != "=":
+            raise ParseError(f"Invalid operator specified for `{search_filter.key.name}`")
+
+        if not isinstance(
+            search_filter.value.value, str
+        ) or search_filter.value.value.lower() not in ["true", "false"]:
+            raise ParseError(f"Could not parse value for `{search_filter.key.name}`")
+
+        operator = "=" if search_filter.value.value.lower() == "true" else "!="
```

Comment:
Thinking on this more.  Let's always use the `=` operator.  The value `true` or `false` will change based on user input.

Reply:
Sorry ignore this.  This is wrong.  I wasn't thinking clearly.